vite.config.ts

```diff
@@ -0,0 +1,45 @@
+/** @type {import('vite').UserConfig} */
+import path from "path"
+import { mergeAlias } from "vite"
+
+export default {
```

Comment:
If we have this file, does that mean we can remove the config that's inside Storybook (which should be the same)?